### PR TITLE
Judicial System: Rewrites pdf generation

### PR DESCRIPTION
**Pull request by gudjong**
# PDF generation without headless browser

## What

Rewrites PDF generation using pdfkit.

## Why

Using pug and puppeteer caused problems with Docker deployments.

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against master before asking for a review

### CODE REVIEW

File: apps/judicial-system/api/src/app/modules/case/case.pdf.ts [hunk 1]
```diff
@@ -57,114 +62,315 @@ export const formatConclusion = (existingCase: Case) => {
...
-    caseFacts: existingCase.caseFacts,
-    witnessAccounts: existingCase.witnessAccounts,
-    investigationProgress: existingCase.investigationProgress,
-    legalArguments: existingCase.legalArguments,
-    prosecutorName: existingCase.prosecutor.name,
-    prosecutorTitle: existingCase.prosecutor.title,
+  const doc = new PDFDocument({
+    size: 'A4',
   })
+  const stream = doc.pipe(new streamBuffers.WritableStreamBuffer())
```
Comment (oddsson) [inline apps/judicial-system/api/src/app/modules/case/case.pdf.ts:85]: This is an interesting way to create text :D

**Review by oddsson — COMMENTED** [review]
Looks awesome! I see that you are not deleting the PUG files you created earlier. Do we want to keep those files?

**Comment by gudjong** [comment]
> Looks awesome! I see that you are not deleting the PUG files you created earlier. Do we want to keep those files?

No, we want to delete them, but not yet. The build is copying the folder that contains them and would fail if we delete it too early.

**Review by pshomov — APPROVED** [review]

**Comment by pshomov** [comment]
👍🏻

**Review by oddsson — APPROVED** [review]

**Pull request merged**